portfolio/src/main/webapp/index.html

```diff
@@ -62,6 +62,82 @@ <h2>Hello, my name is Christian Liguori.</h2>
       </div>
     </div>
 
+    <!-- Project Showcase -->
+    <div id="project-showcase-container">
+      <div class="container">
+        <div class="row">
+          <div class="col s10 offset-s1">
+            <h3>Here are some of my previous projects.</h3>
+
+            <!-- Project 1 -->
+            <div class="project-card">
+              <a href="https://github.com/cl2112/CPS_3320_Project_3">
+                <h4>Get Involved NJ App</h4>
+              </a>
+              <h5><i>Python</i></h5>
```

Comment:
`<i>` is technically only meant to offset text from other normal surrounding text and you probably rarely need it (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i). Use `font-style: italic;` in CSS if all you want is to italicize this text.